styles.css

```diff
@@ -63,14 +71,33 @@
 
 .navlinks,
 .headImage{
-	background-image:url("headImage.jpg");
-	height:400px;
+	height:700px;
 	margin-top:80px;
 	z-index:3;
 	background-size:cover;
-	background-repeat: no-repeat;
+	background: url("headImage.jpg") no-repeat;
+}
+
+.buttonHolder{
+	height:100px;
+	position:relative;
+	top:60px;
+}
 
-	/*background-position: left bottom;*/
+.buttonHolder div{
+	width:30%;
+	height:100px;
+	background-color:gray;
+	margin:0 auto;
+	position:relative;
+	opacity:0.7;
+}
+
+.buttonHolder div h2{
+	margin:0 auto;
```

Comment:
is `margin: 0 auto` working? You can try `text-align: center` in the parent element

Reply:
margin was doing nothing & has been removed